em++

```diff
@@ -5,12 +5,16 @@
 
 import sys
 
+from tools.toolchain_profiler import *
+
 sys.argv += ['--emscripten-cxx']
 
 if sys.version_info.major == 2:
   import emcc
   if __name__ == '__main__':
+    ToolchainProfiler.record_process_start()
     emcc.run()
+    exit(0)
```

Comment:
If not profiling, it will literally be `sys.exit`, as it comes from this line https://github.com/kripken/emscripten/pull/4491/files#diff-5127fab9259fbcb028301023272e9085R221. When profiling, it will be https://github.com/kripken/emscripten/pull/4491/files#diff-5127fab9259fbcb028301023272e9085R221.